src/media/js/views/search.js

```diff
@@ -208,9 +208,16 @@ define('views/search',
             if (query) break;
         }
 
-        if (query && queryParam === 'author') {
-            pageTypes += ' leaf';
-            utilsLocal.headerTitle(gettext('Developer Listing'));
+        if (query) {
+            switch (queryParam) {
+                case 'author':
+                    pageTypes += ' leaf';
```

Comment:
you should be building your pageTypes along the way, don't think it should have all three elements at the beginning